### PR TITLE
Add param for alpha docker images

### PR DESCRIPTION
**Pull request by Gourav2906**
Add is an alpha flag to set alpha fluentD image

### CODE REVIEW

File: deploy/kubernetes/setup.sh [hunk 1]
```diff
@@ -182,9 +190,10 @@ kubectl -n $NAMESPACE create secret generic sumologic \
   --from-literal=endpoint-logs=$ENDPOINT_LOGS \
   --from-literal=endpoint-events=$ENDPOINT_EVENTS
 
-echo "Applying deployment 'fluentd'..."
+echo "Applying deployment 'fluentd'... $release"
 curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
```
Comment (rvmiller89) [inline deploy/kubernetes/setup.sh]: could we also change this URL? Now that we automatically tag our releases, we can make the URL like:

curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/v$release/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \

Note:  requires "v" added before $release

File: deploy/kubernetes/setup.sh [hunk 2]
```diff
@@ -182,9 +190,10 @@ kubectl -n $NAMESPACE create secret generic sumologic \
...
 sed 's/\$NAMESPACE'"/$NAMESPACE/g" | \
+sed 's/\0.0.0'"/$release/g" | \
```
Comment (samjsong) [inline deploy/kubernetes/setup.sh]: I think we'd eventually want to update the master yaml to also reflect the latest stable version, in which case we'd have to remember to change it here as well, not sure how I feel about that

Comment (rvmiller89) [reply deploy/kubernetes/setup.sh]: Agreed, I'm going to assume that we will bump the version in the master Yaml every so often.  Could we try to replace this based on a regex search, or at least looking for `image: sumologic/kubernetes-fluentd:` ?

**Comment by Gourav2906** [comment]
Messed up here , created new PR 
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/113

**Pull request closed without merging**